catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/config/JdbcConfig.java

```diff
@@ -70,6 +70,13 @@ public class JdbcConfig extends Config {
           .checkValue(value -> value > 0, ConfigConstants.POSITIVE_NUMBER_ERROR_MSG)
           .createWithDefault(10);
 
+  public static final ConfigEntry<Boolean> TEST_ON_BORROW =
```

Comment:
Can we remove this config and add a retry mechanism in the code?

Reply:
I'm afraid not. The retry logic for the connection in the connection pool is complex and difficult to manage. Furthermore, it's not recommended.